lib/Acho.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const CONST = require('./Constants')
+const DEFAULT = require('./Default')
+
+function Acho (opts = {}) {
```

Comment:
Happened the same here, regular arrow functions don't have this so this logic breaks if it is not a regular function...